### ISSUE TITLE
shot number not updating

### ISSUE THREAD

**Issue by lamorton** [Fusion-Data-Platform/fdp#60]
When I type in a new shot number, I get data from the first shot that I looked at:

![image](https://user-images.githubusercontent.com/23484003/51346506-7778ca00-1a63-11e9-8d71-e5d7cffb644f.png)

For reference, this is what I did to get started:
`
module load nstx/mdsplus_stable
setenv PYTHONPATH /u/lmorton/fdp
setenv PATH /p/fdp/anaconda/bin:$PATH
`
$PATH looks like:
`/p/fdp/anaconda/bin:/usr/pppl/freetds/0.64_64bit/bin:/usr/pppl/java/x64/jdk1.6.4`
$PYTHONPATH:
`/u/lmorton/fdp`
module list:
`  1) flexlm/common             4) nstx/mdsplus_idl_stable   7) nstx/epics
  2) flexlm/11.14.1.2          5) java/v1.6                 8) freetds/0.64_64bit
  3) idl/8.4                   6) nstx/mdsplus_treedefs     9) nstx/mdsplus_stable
`

**Comment by lamorton** [comment]
FWIW: 
![image](https://user-images.githubusercontent.com/23484003/51346832-2f0ddc00-1a64-11e9-9fbc-6585cb912d03.png)

**Comment by drsmith48** [comment]
can you try again with:
module load anaconda3
module load nstx/mdsplus_alpha
(don't use /p/fdp/anaconda)

**Comment by lamorton** [comment]
Okay, fresh session with startup commands:
```
module load anaconda3
module load nstx/mdsplus_alpha
setenv PYTHONPATH /u/lmorton/fdp:$PYTHONPATH
```
![image](https://user-images.githubusercontent.com/23484003/51347333-63ce6300-1a65-11e9-988c-4f15f67190cb.png)

**Comment by drsmith48** [comment]
oh, right, pymssql is not in the python/anaconda modules.  I should ask the helpdesk to install pymssql, but a quick fix is for you to clone my (nearly up-to-date) conda environment:
`$ conda create -n <your conda env name> --clone /u/drsmith/nstxusr/conda/envs/py37`
That will give you a good python 3.7 environment, but it doesn't fix your original bug.  I reproduced your bug, so it's an FDP problem (not python/mdsplus as I suggested earlier).  I'll investigate.
